docs/body.md

```diff
@@ -248,8 +252,8 @@ Choose that option and you'll be presented with a dialog asking for the
 following inputs.
 
 * Device To Create - non-empty, non-ip, non-dns, unique name to use for this device in Zenoss. ''See note below''.
-* Auth URL - A keystone URL, such as http://\<hostname\>:5000/v2.0/
-* Username, Password/API Key, Project/Tenant ID - *Administrative* credentials to your Zenoss instance.
+* Auth URL - A keystone URL, such as http://\<hostname\>:5000/v2.0/ for Keystone v2, or http://\<hostname\>:5000/v3/ for Keystone v3.
```

Comment:
I thought I tried sans version and got an error.  Let me double check we don't have a bug, and I'll update it.  Fair point about the versioning.  I'll clean that up.